crypto/bip39_english.h

```diff
@@ -21,7 +21,9 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
```

Comment:
`crypto/bip39_english.h` should be changed to `crypto/bip39_english.c` and added to the build. The `#include bip39_english.h` in `bip39.c` should then be removed.

Reply:
Or you can create a new `bip39_english.h` file and move the `extern` declaration into that file.

Reply:
done in [d97d3d3](https://github.com/trezor/trezor-firmware/pull/2312/commits/d97d3d3b4c7b736a308354eeeb3d05ca2f7e50df) and [eac8923](https://github.com/trezor/trezor-firmware/pull/2312/commits/eac89230d0b553fcbe1cc589ee333b34a1bc70f6)